production.Dockerfile

```
@@ -0,0 +1,60 @@
+# Keeps Python from buffering stdout and stderr to avoid situations where
+# the application crashes without emitting any logs due to buffering.
+ENV PYTHONUNBUFFERED=1
+
+# Set Django settings module
+ENV DJANGO_SETTINGS_MODULE=todo_project.settings.production
+ENV ENV=PRODUCTION
+
+WORKDIR /app
+
+# Install CA certificates needed for TLS connections to MongoDB Atlas
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)